src/proj.rs

```diff
@@ -673,11 +673,17 @@ impl Proj {
         // This signals that we wish to project geodetic coordinates.
         // For conversion (i.e. between projected coordinates) you should use
         // PJ_XY {x: , y: }
-        let coords = PJ_LP { lam: c_x, phi: c_y };
+        // We also initialize z and t in case libproj tries to read them.
+        let coords = PJ_LPZT {
+            lam: c_x,
+            phi: c_y,
+            z: 0.0,
+            t: f64::INFINITY,
```

Comment:
Curious, why did you choose infinity here?

Reply:
It's what libproj does in `proj_transform_generic` (or another function with a similar name).